packages/fetchai/skills/simple_oracle/handlers.py

```diff
@@ -175,7 +175,7 @@ def _request_init_transaction(self, ledger_id: str, tx_receipt: JSONLike) -> Non
                         "label": "OracleContract",
                         "init_msg": {"fee": str(strategy.initial_fee_deploy)},
                         "gas": strategy.default_gas_deploy,
-                        "amount": 0,
+                        "amount": 1,
```

Comment:
Why this change?

Reply:
This was the easiest way to make sure some funds (along with the denomination) were sent to the contract. Otherwise, we would have needed to send the denomination as an extra string parameter. So this is actually what fixed the test!